cylc/flow/task_job_mgr.py

```diff
@@ -842,15 +874,20 @@ def _prep_submit_task_job_error(self, suite, itask, dry_run, action, exc):
 
     def _prep_submit_task_job_impl(self, suite, itask, rtconfig):
         """Helper for self._prep_submit_task_job."""
-        itask.task_owner = rtconfig['remote']['owner']
-        if itask.task_owner:
-            owner_at_host = itask.task_owner + "@" + itask.task_host
+        # itask.platform is going to get boring...
+        platform = itask.platform
+        host = get_host_from_platform(platform)
+        owner = platform['owner']
+
+        if owner:
+            owner_at_host = f"{owner}@{host}"
```

Comment:
Do we have an `owner` field for platforms?

Reply:
Aye

Reply:
Should we have an `owner` field (we don't support `--owner` logic any more so this does nothing right?).

Reply:
I'll remove it.

Reply:
I'll remove it.
@oliver-sanders  - I have created a PR to this PR's branch at https://github.com/wxtim/cylc/pull/12 which I believe answers both of your remaining comments.